src/parsers/js/transformers/eslint2/index.js

```diff
@@ -0,0 +1,23 @@
+import {defineRule, runRule} from '../../utils/eslintUtils';
+import pkg from 'eslint2/node_modules/eslint/package.json';
+
+const ID = 'eslint-v2';
+const name = 'ESLint v2'
+
+export default {
+  id: ID,
+  displayName: name,
+  version: pkg.version,
+  homepage: pkg.homepage,
+
+  defaultParserID: 'acorn-to-esprima',
+
+  loadTransformer(callback) {
+    require(['eslint2'], callback);
+  },
+
+  transform({ eslint, rules, sourceCode }, transformCode, code) {
+    defineRule(rules, transformCode);
+    return runRule(code, eslint, sourceCode);
```

Comment:
@hzoo - I needed to use `parseNoPatch` since the monkey-patching doesn't work within the context of a browser (or really any non-Nodejs environment), as `Module._resolveFilename` `new Module` and ` Module._nodeModulePaths` are Node-isms. It assumes a Node.js environment.